### PR TITLE
Only wait for JHPORT b/f assigning FIP if needed.

### PR DESCRIPTION
**Pull request by garloff**
This was required on OTC, and we can still force the waiting by setting the FIPWAITPORTDEVOWNER environment.

Signed-off-by: Kurt Garloff <kurt@garloff.de>

### CODE REVIEW

File: api_monitor.sh [hunk 1]
```diff
@@ -1836,11 +1836,12 @@ createFIPs()
   # Not needed on OTC, but for most other OpenStack clouds:
   # Connect Router to external network gateway
   ostackcmd_tm NETSTATS $NETTIMEOUT neutron router-gateway-set ${ROUTERS[0]} $EXTNET
-  # Actually this fails if the port is not assigned to a VM yet
-  #  -- we can not associate a FIP to a port w/o dev owner
-  # So wait for JHPORTS having a device owner
-  #echo "Wait for JHPorts: "
-  waitResources NETSTATS JHPORT JPORTSTAT JVMSTIME "NONNULL" "NONONO" "device_owner" $NETTIMEOUT neutron port-show
+  if test -n "$FIPWAITPORTDEVOWNER"
```
Comment (garloff) [inline api_monitor.sh:1840]: a `; then` is missing here, sigh.
(Manual cut'n'paste is error-prone.)

**Comment by garloff** [comment]
Reviews highly welcome!

**Comment by garloff** [comment]
May I ask for a review?
@fkr, @jnull, @chschilling ?
TIA!

**Review by fkr — APPROVED** [review]
LGTM

**Pull request merged**